packages/docs-theme/src/components/interfaceTable.tsx

```diff
@@ -31,79 +30,91 @@ function propTag(intent: Intent, title: string, ...children: React.ReactNode[])
     );
 }
 
-const renderPropRow = (prop: IInheritedPropertyEntry) => {
-    const { documentation, inheritedFrom, name, optional } = prop;
-    const { default: defaultValue, deprecated, internal } = prop.tags;
-
-    if (internal) {
-        return undefined;
+function renderPropType(prop: ITsProperty | ITsMethod) {
+    if (isTsProperty(prop)) {
+        const formattedType = prop.type.replace(/\b(JSX\.)?Element\b/, "JSX.Element");
+        return (
+            <code className="docs-prop-type">
```

Comment:
(out of scope for this PR)

do we have any namespacing strategy here? should these be `bp-`?

Reply:
oh dang that's a great question! current namespacing strategy is `docs-`. didn't we just kill the `bp-` prefix in #1906? how about `pt-docs-` since it's a public package?

Reply:
well yeah we killed `bp-` for our library but this is a little different since it's documentalist-based...

I think `pt-docs-` would work fine though

Reply:
cool i'll migrate to that in an upcoming PR. got a few things i want to do for docs-theme 2.0.